src/Codeception/Subscriber/ErrorHandler.php

```diff
@@ -39,11 +44,16 @@ public function handle(SuiteEvent $e)
             $this->errorLevel = eval("return {$settings['error_level']};");
         }
         error_reporting($this->errorLevel);
+
+        if (self::$initialized) {
```

Comment:
it was done by the same manner as `self::$stopped`. Can we make `$stopped` to be instance variable as well?
Won't it break anything?